package Egametang;

message RpcChannelTestRequest {
	required int32 num = 1;
};

message RpcChannelTestResponse {
	required int32 num = 1;
};

service RpcChannelTestService {
  rpc Echo(RpcChannelTestRequest) returns (RpcChannelTestResponse);
};